index.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./css/style.css">
+    <title>Capstone Project</title>
+</head>
+<body>
+    <!--Header Section-->
+    <section id="header__section" class="header__section">
+        <header class="header__container">
+            <a href="#header__section" class="logo__image">
+                <img src="./images/logo.png" alt="logo_image">
+            </a>
+            <nav class="header__navbar">
+                <i id="menuOpen" class="fa-solid fa-bars"></i>
+                <i id="menuClose" class="fa-solid fa-times"></i>
+                <ul class="navbar__container">
+                    <li class="navbar__list__item ">
+                        <a href="" class="navbar__list__link active">About</a>
+                    </li>
+                    <li class="navbar__list__item">
+                        <a href="" class="navbar__list__link">Program</a>
+                    </li>
+                    <li class="navbar__list__item">
+                        <a href="" class="navbar__list__link">Join</a>
+                    </li>
+                    <li class="navbar__list__item">
+                        <a href="" class="navbar__list__link">Sponsor</a>
+                    </li>
+                    <li class="navbar__list__item">
+                        <a href="" class="navbar__list__link">News</a>
+                    </li>
+                    <li class="navbar__list__item campaign__list">
+                        <a href="" class="navbar__list__link campaign__link">CC Campaign</a>
+                    </li>
+                </ul>
+            </nav>
+        </header>
```

Comment:
According to the design it seems like you missed the top short bar section with social media links, logout, and language links aligned to the right. Kindly I suggest you should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including:
Colors.
Typography: font face, size and weight (we suggest using [Lato](https://www.latofonts.com/) as it is a free font similar to the one used in the original design).
Layout: composition and space between elements.

> ## Your Design
>
> <img width="1440" alt="Screenshot 2023-06-28 at 4 26 55 PM" src="https://github.com/Feliperto/Capstone-Project/assets/91237525/317fa433-caab-4a0b-8e48-52084b12ce87">
>
> ## The required design
>
><img width="1440" alt="Screenshot 2023-06-28 at 4 27 40 PM" src="https://github.com/Feliperto/Capstone-Project/assets/91237525/b74412fa-1533-4d30-8dff-ae235e859ec8">